ImplementationGuide/markdown/UebergreifendeFestlegungen/UebergreifendeFestlegungen_Suchparameter.md

```diff
@@ -59,42 +71,33 @@ Diese Suche gibt alle Condition-Ressourcen zurück zum Client, welche innerhalb
 
 ### Reference
 
-Der Modifier `:identifier` MUSS für alle spezifizierten Suchparameter vom Typ 'Reference' unterstützt werden.
+Der Modifier `:identifier` KANN für alle spezifizierten Suchparameter vom Typ 'Reference' unterstützt werden.
 
-Der [type] Modifier MUSS für alle spezifizierten Suchparameter vom Typ 'Reference' unterstützt werden.
+Der Modifier :identifier MUSS implementiert werden, wenn die entsprechende Reference eine 1..1-Kardinalität oder ein MS-Flag auf Reference.identifier hat.
 
-**Beispiele**:
+Dies gilt beispielsweise für Encounter.account - also die Referenz zwischen ISiKKontaktGesundheitseinrichtung und ISiKAbrechnungsfall. Encounter MÜSSEN anhand der Fallnummer gesucht werden können, ohne dass Clients Zugriffsberechtigungen auf Accounts haben müssen, bzw. ohne dass Account zwingend implementiert/referenziert werden muss. Der Suchabruf erfolgt entsprechend dann nur über die logische Referenz.
+
+Begründung: Die Unterstützung dieser Suchparameter-Typen ist essentiell für Abfragen mit [Chaining](https://hl7.org/fhir/r4/search.html#chaining) und [Reverse Chaining](https://hl7.org/fhir/r4/search.html#has). Innerhalb der Spezifikation ist für jedes Datenobjekt spezifiziert weshalb eine solche Abfrage versorgungsrelevant ist.
```

Comment:
issue: m.M.n. hat der Suchparameter :identifier erstmals mit Chaining nichts zu tun - damit ist sogar kein Chaining möglich (cf. https://www.hl7.org/fhir/R4/search.html#reference) Der Modifier hat aus meiner SIcht eher den oben beschriebenen Zweck - also Suche nach Identifiern von assoziierten Ressourcen zu ermöglichen falls die assoziierten Ressourcen im System als eigenständige Entitäten nicht existieren

Reply:
@simoneOnFhir , @alexey-tschudnowsky hat recht. Zumindest di Begründung an dieser Stelle ist einfach falsch.
Muss auch in Stufe 4 nochmal geändert werden.